### PR TITLE
train example

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
Hi, i know about ternary operator, thats bad practice to put in `return` statement. But it really short code.

### CODE REVIEW

**Review by mgrinko — CHANGES_REQUESTED** [review]
Watch this https://youtu.be/tpB5vtFGcrg

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return (40 * days) - 50;
+  } else if (days >= 3) {
+    return (40 * days) - 20;
+  }
```
Comment (illiarerun) [inline src/calculateRentalCost.js:35]: [Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item. You should create variables for main values - 20, 50, 40 they shouldn't be just magic numbers

Comment (illiarerun) [reply src/calculateRentalCost.js:35]: Also, checklist 3rd item, why do you have else if after a return statement? Code won't execute past this, use plain if construction

**Review by illiarerun — CHANGES_REQUESTED** [review]
Nice start, now let's work on some fixes:

**Review by illiarerun — APPROVED** [review]
Gj, but I wouldn't use switch statement here

**Pull request closed without merging**